main.go

```diff
@@ -26,19 +26,25 @@ func main() {
 		upstream string
 		branch   string
 		username string
-		password string
+		token    string
 	)
 
 	flag.StringVar(&upstream, "upstream", "https://github.com/okkur/gitor.git", "specifies upstream")
 	flag.StringVar(&branch, "branch", "master", "specifies branch")
 	flag.StringVar(&username, "username", username, "specifies username")
-	flag.StringVar(&password, "password", password, "specifies password")
+	flag.StringVar(&token, "token", token, "specifies token/password")
 	flag.Usage = usage
 
 	flag.Parse()
-	command := flag.Args()[0]
-	if command == "update" {
-		update(upstream, branch, username, password)
+
+	switch {
+	case len(os.Args) > 2 && os.Args[2] == "update":
```

Comment:
Overall a nice solution. So there are a few options available to make this solution cleaner.
As we are already using the flag package, we can also reuse that.
So firstly yes os.Args[2] with the len check works good.

There are 2 functions available for Args in the flag package:
* flag.Args()
* flag.Arg()

So we can either use:
```
command := flag.Arg(0)
switch {
case command == "update":
  update(upstream, branch, username, token)
default:
  usage()
}
```

Or something like:
```
commands := flag.Args()
switch {
case len(commands) > 0 && commands[0] == "update":
  update(upstream, branch, username, token)
default:
  usage()
}
```

This would prevent us from using both implementations os.Args and flag.Args. With the usage of flag.Arg() also make the code cleaner.